ci_environment/docker/recipes/default.rb

```diff
@@ -18,10 +18,15 @@
   action :add
 end
 
-docker_pkg = 'docker-engine'
-# TODO docker_pkg += "-#{node['docker']['version']}" if node['docker']['version']
-
-package %W(linux-image-extra-#{`uname -r`.chomp} lxc #{docker_pkg})
+package %W(
+  linux-image-extra-#{`uname -r`.chomp}
```

Comment:
The package is extra kernel modules for uncommon devices etc that I'm 99% sure you don't need. e.g http://packages.ubuntu.com/precise/amd64/linux-image-extra-3.2.0-23-virtual/filelist . I've never needed this on GCE etc.

Also because this runs prior to the reboot `uname -r` is going to return the kernel version that the packer base image is using, and not the installed vivid kernel.

You're probably safe just to remove this.